src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -24,15 +24,47 @@
 
     static $phpunitConfig = null;
     if (null === $phpunitConfig) {
-        $opt = min(array_search('-c', $opts = array_reverse($argv), true) ?: INF, array_search('--configuration', $opts, true) ?: INF);
         $phpunitConfigFilename = null;
-        if (INF !== $opt && isset($opts[$opt - 1])) {
-            $phpunitConfigFilename = $opts[$opt - 1];
```

Comment:
checking for a directory here, and try file_exists phpunit.xml or .dist to resolve the config file path isnt sufficient?